README.md

```diff
@@ -102,8 +102,12 @@ Perform the following commands on the terminal and write them down for review,
   - Checkout new branch
   - Merge branch into another
   - See commit history
-  - Rebase and Squash
   - Write a command to clone [this](https://github.com/canopas/web-developer-roadmap-2023) repo
+  - Stash changes on the current branch, switch to another branch to apply a hotfix, and then apply the stashed changes back.
```

Comment:
It's about specifying results, rather than actions. I think the steps shouldn't be involved as per the roadmap context.

Reply:
Git commands in the roadmap were not designed based on the roadmap's use cases. It will be the scenario developers will experience during actual projects. 

From my perspective, this will be helpful to them in the following usecase,
They made some changes and don't want to push those changes. At the same time, they need to make some changes in another branch. In this case, these commands will be very useful. 

They need to perform the given scenario and write those commands in this practical